MosipNexus/api/main.py

```
@@ -0,0 +1,529 @@
+         description="Returns ChromaDB collection chunk counts and active session count.")
+def health() -> HealthResponse:
+    collections: dict[str, Any] = {}
+    try:
+        docs_store, community_store = _retriever._get_stores()
+        collections["mosip_docs"] = docs_store._collection.count()
+        collections["community"] = community_store._collection.count()
+        if _retriever._github_store:
+            collections["github"] = _retriever._github_store._collection.count()
+        if _retriever._code_store:
+            collections["source_code"] = _retriever._code_store._collection.count()
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)